ospo-mindmap/Content/ospomindmap.md

```diff
@@ -75,6 +75,7 @@ development team of the Linux Project
   - Adopt existing tools
       - Commercial tools
       - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
+      - [Linux Foundation's Automate Compliance Tooling](https://automatecompliance.org/)
```

Comment:
fix indentation to show as part of the open source tooling section:

```
  - Adopt existing tools
      - Commercial tools
      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
             - [Linux Foundation's Automate Compliance Tooling](https://automatecompliance.org/)
```

Reply:
yes, it is better under Open Source tools. 
Thanks Ana!